.github/workflows/security-scans.yml

```
@@ -114,3 +114,34 @@ jobs:
           echo "::endgroup::"
           # Docs on how to configure exceptions: https://hub.armosec.io/docs/exceptions
           kubescape scan framework ${{ matrix.kubescape-framework }} -v -t ${{ env.FAILURE_PERCENTAGE }} --exceptions ./.github/.kubescape/exceptions.json --controls-config ./.github/.kubescape/controls-inputs.json ${{ env.CHART_TO_SCAN }} --enable-color
+  
+  create_issue:
+    name: Create GitHub Issue
+    runs-on: ubuntu-20.04
+    needs: security-scans
+    if: ${{ failure() }}
```

You don't need the curly brackets here
```suggestion
    if: failure()
```